source/documentation/appendix-docs/conda.md

```diff
@@ -1,89 +1,73 @@
-# Conda Package Management
+# Package management
 
-## Unified package management for both R and Python
+## Introduction
 
-### Faster builds
+A key element of ensuring analysis is reproducible is maintaining a record of the versions of packages used to produce said analysis. There are a number of package management tools available for both R and Python. Within the MoJ we recommend primarily using [Conda](#Conda) for both languages.
 
-CRAN mirrors do not provide linux compiled binaries
-for packages. This means a long wait when doing `install.packages` both in an R
-Studio session and when running a Docker build for a `shiny` application.
+## R
 
-### Cross Language
+### Packrat
 
-Both `dbtools` and `s3tools` rely on Python packages through the `reticulate` R
-to Python bridge. `packrat` only handles R dependencies, this means that
-`packrat` is not enough to reproducibly and reliably manage all of your
-application's dependencies.
+Packrat is the most well-known package management tool for R. There's more information about it here: https://rstudio.github.io/packrat/
 
-`conda` supports managing both Python and R dependencies in a single
-`environment`. It can make sure all of these libraries are compatible with each
-other.
+It has some significant downsides. It can be quite temperamental, and difficult to debug when things go wrong - in the earlier days of the Analytical Platform, the majority of support issues related to getting Packrat working. Furthermore, CRAN mirrors do not provide linux compiled binaries for packages. This means a long wait when doing `install.packages` both in an RStudio session and when running a Docker build for a `shiny` application.
 
-## Installing Packages
+### Renv
 
-If you need to find a package name you can use the [anaconda search] to find the
-name. Run `conda install PACKAGENAME` in the `Terminal` tab to install it. For
-more advanced usage have a look at the [conda cheat sheet].
+[Renv](https://rstudio.github.io/renv/articles/renv.html) is a newer package billed as "Packrat 2.0". This has a number of improvements over Packrat, in the speed of download and reduction of issues of 00LOCK files that often plague Packrat. However, it is still not able to deal with OS-level dependencies, so Conda is still preferred.
 
-### R
+## Conda
 
-Most CRAN (around 95%) are available through `conda`, they have the same name as
+Conda is a unified package management system for both R and Python. `conda` supports managing both Python and R dependencies in a single `environment`. It can make sure all of these libraries are compatible with each other.
+
+A key example within Analytical Services: both `dbtools` and `s3tools` rely on Python packages through the `reticulate` R-to-Python bridge. `packrat` only handles R dependencies; this means that `packrat` is not enough to reproducibly and reliably manage all of your application's dependencies.
+
+### Installing Packages
```

Comment:
It would probably be helpful to explicitly say that using conda means that users are no longer advised to do `install.packages(blah)` in here (as well as in the table under Examples).

Reply:
good point. Done